graphql_compiler/tests/test_ir_generation.py

```diff
@@ -2646,6 +2646,12 @@ def test_no_op_coercion_inside_fold(self):
 
         check_test_data(self, test_data, expected_blocks, expected_location_types)
 
+    def test_no_op_coercion_noninvertible_hints(self):
+        # The type where the coercion is applied is already Entity, so the coercion is a no-op.
```

Comment:
copy-pasta, also this should be unnecessary once you move the non-invertible test to `test_ir_generation_errors.py`